yaksh/models.py

```diff
@@ -383,6 +383,20 @@ def get_failed_students(self, course):
                 course=course, passed=False
             ).values_list("user", flat=True).distinct().count()
 
+    def get_answerpaper_status(self, user, course):
```

Comment:
```AnswerPaper``` model already has ```status``` attribute, so the name ```get_answerpaper_status``` is not proper here.

Reply:
Now `get _answerpaper_status` will return answerpaper status i.e in progress or completed.